apps/web/src/components/NavMenu/MainNavMenu.tsx

```diff
@@ -72,82 +72,98 @@ const MainNavMenu = () => {
         )
       }
     >
-      <NavMenu.List type="main">
-        <HomeLink
-          href={paths.home()}
-          label={intl.formatMessage(navigationMessages.home)}
-        />
-        {showRoleSwitcher ? (
-          <>
-            <NavMenu.Item className={borderItem({ borderRight: true })}>
+      <div
+        role="group"
+        aria-label={intl.formatMessage({
+          defaultMessage: "Main navigation",
+          id: "SY1LIh" /*  TODO: translate & update default message */,
+          description: "Label for the main navigation",
+        })}
+      >
```

Comment:
I'm not sure that the correct answer here is because I have always been confused by this part of our nav but, I don't know main nav make sense (or if it being its own group even does).

I think we need to either:

 1. Try to group it with the other elements or,
 2. Find a different name for it